modules/adheseBidAdapter.js

```diff
@@ -20,11 +22,15 @@ export const spec = {
     if (validBidRequests.length === 0) {
       return null;
     }
+
     const { gdprConsent, refererInfo } = bidderRequest;
 
+    const adheseConfig = config.getConfig('adhese');
     const gdprParams = (gdprConsent && gdprConsent.consentString) ? { xt: [gdprConsent.consentString] } : {};
     const refererParams = (refererInfo && refererInfo.referer) ? { xf: [base64urlEncode(refererInfo.referer)] } : {};
-    const commonParams = { ...gdprParams, ...refererParams };
+    const globalCustomParams = (adheseConfig && adheseConfig.globalTargets) ? adheseConfig.globalTargets : {};
+    const commonParams = { ...gdprParams, ...refererParams, ...globalCustomParams };
+    const vastContentAsUrl = (adheseConfig && adheseConfig.vastContentAsUrl !== 'undefined') ? adheseConfig.vastContentAsUrl : true;
```

Comment:
better to make sure it's always a bool - so we don't break the request because if someone defines some fishy type in the config